content/staging-area.md

```diff
@@ -161,61 +161,134 @@ We give two examples and the instructor can pick one or both:
 
 ## Staging area commands
 
-With the **staging area**, there is one middle point before commits.
+The staging area creates a layer between the working directory and the next
+commit. This adds some complexity but also adds more flexibility to selectively
+prepare commits since **you can modify and stage several times before committing**.
+
+**git add** stages/prepares for the next commit:
+```text
+                git add
+ [project*]  <------------  project*
+                               ^
+                               | modify with editor
+                               |
+  HEAD            .         project
+    |             .
+    |             .
+    |             .
+  HEAD~1          .
+    |             .
+    |             .
+    |             .
+                  .
+  .git            .     working directory
+(history)         .      (files we see)
+```
 
-```{figure} img/staging-basics.svg
-:alt: Staging basics
-:width: 100%
+**git commit** creates a new commit:
+```text
+
+  HEAD            .         project
+    |             .
+    |             .
+    |             .
+  HEAD~1          .
+    |             .
+    |             .
+    |             .
+  HEAD~2          .
+    |             .
+    |             .
+    |             .
+                  .
+  .git            .     working directory
+(history)         .      (files we see)
 ```
 
-Files can be untracked, modified, staged, or committed, and
-we have a variety of commands to go between states:
-
-```shell
-$ git add <path>       # stages all changes in file
-$ git add -p <path>    # stages while letting you choose which lines to take
-$ git commit           # commits the staged change
-$ git diff             # see **unstaged** changes
-$ git diff --staged    # see **staged** changes
-$ git rm               # removes a file
-$ git reset            # unstages staged changes
-                       # in latest Git: git restore --staged <path>
-$ git checkout <path>  # check out the latest staged version ( or committed
-                       # version if file has not been staged )
-                       # in latest Git: git restore <path>
+**git reset --soft HEAD~1** (move HEAD back by one but keep changes and stage
+them) would do the opposite of **git commit**.
+
+Going back to the last staged version:
+```text
+              git restore
+ [project*]  ------------>  project*
```

Comment:
When I first read this, I tried to understand how the diagram represented that.  Then I realized it didn't and was a different thing.  Perhaps add `**git restore**` (bold) to line 211 (right before the diagram) to make it clear that's what it is about.